tmt/utils.py

```diff
@@ -269,9 +303,14 @@ def debug(self, key, value=None, color=None, shift=1, level=1, err=False):
         if self.opt('debug') >= level:
             echo(self._indent(key, value, color, shift), err=err)
 
-    def _run(
-            self, command, cwd, shell, env, log, join=False, interactive=False,
-            timeout=None):
+    def _process_run(self, interactive, command, cwd, shell, environment, stdout, stderr):
+        process = subprocess.Popen(command, cwd=cwd, shell=shell, env=environment, stdin=subprocess.DEVNULL, stdout=stdout, stderr=stderr)
```

Comment:
IMHO this stdin breaks interactive mode
reproduer: `tmt test create -t beakerlib a`. Add `bash` somewhere
```
    execute
        how: tmt
        progress: /a [1/1]    finish
        summary: 0 tasks completed
Traceback (most recent call last):
  File "/home/lzachar/.virtualenvs/tmt/bin/tmt", line 7, in <module>
    exec(compile(f.read(), __file__, 'exec'))
  File "/home/lzachar/gits/tmt/bin/tmt", line 8, in <module>
    tmt.cli.main()
  File "/home/lzachar/.virtualenvs/tmt/lib/python3.9/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/home/lzachar/.virtualenvs/tmt/lib/python3.9/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/home/lzachar/.virtualenvs/tmt/lib/python3.9/site-packages/click/core.py", line 1259, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/lzachar/.virtualenvs/tmt/lib/python3.9/site-packages/click/core.py", line 1290, in invoke
    return _process_result(rv)
  File "/home/lzachar/.virtualenvs/tmt/lib/python3.9/site-packages/click/core.py", line 1224, in _process_result
    value = ctx.invoke(self.result_callback, value, **ctx.params)
  File "/home/lzachar/.virtualenvs/tmt/lib/python3.9/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/home/lzachar/.virtualenvs/tmt/lib/python3.9/site-packages/click/decorators.py", line 21, in new_func
    return f(get_current_context(), *args, **kwargs)
  File "/home/lzachar/gits/tmt/tmt/cli.py", line 297, in finito
    click_context.obj.run.go()
  File "/home/lzachar/gits/tmt/tmt/base.py", line 1368, in go
    plan.go()
  File "/home/lzachar/gits/tmt/tmt/base.py", line 763, in go
    step.go()
  File "/home/lzachar/gits/tmt/tmt/steps/execute/__init__.py", line 138, in go
    plugin.go()
  File "/home/lzachar/gits/tmt/tmt/steps/execute/internal.py", line 170, in go
    self.execute(test, guest, progress=f"{index + 1}/{len(tests)}")
  File "/home/lzachar/gits/tmt/tmt/steps/execute/internal.py", line 124, in execute
    stdout = guest.execute(
  File "/home/lzachar/gits/tmt/tmt/steps/provision/local.py", line 63, in execute
    return self.run(command, **kwargs)
  File "/home/lzachar/gits/tmt/tmt/utils.py", line 407, in run
    return self._run(
  File "/home/lzachar/gits/tmt/tmt/utils.py", line 340, in _run
    process = self._process_run(interactive=interactive, command=command, cwd=cwd, shell=shell, environment=environment, stderr=stderr, stdout=stdout)
  File "/home/lzachar/gits/tmt/tmt/utils.py", line 304, in _process_run
    process = subprocess.Popen(command, cwd=cwd, shell=shell, env=environment, stdin=subprocess.DEVNULL, stdout=stdout, stderr=stderr)
  File "/usr/lib64/python3.9/subprocess.py", line 951, in __init__
    self._execute_child(args, executable, preexec_fn, close_fds,
  File "/usr/lib64/python3.9/subprocess.py", line 1821, in _execute_child
    raise child_exception_type(errno_num, err_msg, err_filename)
OSError: [Errno 9] Bad file descriptor: '/var/tmp/tmt/run-003/plans/default/discover/default/tests/a'
```

Reply:
Run as `$ tmt run -a provision -h local execute -h tmt -i`